torch/_dynamo/create_parameter_op.py

```diff
@@ -19,6 +19,7 @@
     ),
     impl_aten=lambda self, placeholder: placeholder.set_(self),
     doc=doc,
+    returns_alias=True,
```

Comment:
This one's suspicious: why does the op return an alias if it's not mentioned in the schema?